config/mail.global.php.dist

```diff
@@ -2,20 +2,20 @@
 
 declare(strict_types=1);
 
+use Symfony\Component\Mailer\Transport\SendmailTransport;
+
 return [
 
     /**
      * Dotkernel mail module configuration
-     * Note that many of these options can be set programmatically too, when sending mail messages
-     * actually that is what you'll usually do, these config provide just default and options that remain the same for all mails
+     * Note that many of these options can be set programmatically too, when sending mail messages actually that is
+     * what you'll usually do, these config provide just default and options that remain the same for all mails
      */
-
     'dot_mail' => [
         //the key is the mail service name, this is the default one, which does not extends any configuration
         'default' => [
             //tells which other mail service configuration to extend
             'extends' => null,
```

Comment:
@SergiuBota1 Have you foun any usage of this?
I searched through the entire `mailer` package, but could not find any reference to this.
Also, created a dummy class with an exit message in it's constructor - it did not get executed.
So, I'm fairly sure that we don't need this field either.